src/anemoi/training/config/diagnostics/eval_rollout.yaml

```diff
@@ -71,7 +71,7 @@ log:
     offline: False
     log_model: False
     project: 'Anemoi'
-    entity: ???
+    entity: []
```

Comment:
Is this an unrelated fix?

Reply:
That's a remnant from when missing values were behaving incorrectly but that's resolved now. Sara, can you change this back to `???`